### PR TITLE
Removed unnecessary conditions in op_add and op_mul

### PR DESCRIPTION
**Pull request by ckmadhira**
Summary:
Added scalar function call in op_add and op_mul
Updated undefining of macros in op_quantize
Updated elseif() instead of if() in op_cat

### CODE REVIEW

**Comment by ckmadhira** [comment]
> @pytorchbot label "topic: not user facing"

File: backends/cadence/fusion_g3/operators/op_cat.cpp [hunk 1]
```diff
@@ -37,6 +34,7 @@ Tensor& cat_out(
     exec_aten::ArrayRef<Tensor> tensors,
     int64_t dim,
     Tensor& out) {
+
```
Comment (mcremon-meta) [inline backends/cadence/fusion_g3/operators/op_cat.cpp:37]: please remove new line to make linter happy

File: backends/cadence/fusion_g3/operators/op_quantize.cpp [hunk 2]
```diff
@@ -565,6 +565,7 @@ Tensor& quantize_per_tensor_out(
     int64_t quant_max,
     ScalarType dtype,
     Tensor& out) {
+
```
Comment (mcremon-meta) [inline backends/cadence/fusion_g3/operators/op_quantize.cpp:568]: please remove new line to make linter happy

File: backends/cadence/fusion_g3/operators/op_quantize.cpp [hunk 3]
```diff
@@ -802,6 +802,6 @@ Tensor& quantize_per_token_out(
 }
 
 } // namespace native
-} // namespace FusionG3
+} // namespace G3
 } // namespace impl
 } // namespace cadence
```
Comment (mcremon-meta) [inline backends/cadence/fusion_g3/operators/op_quantize.cpp:807]: Nit: missing new line at end of file

**Review by mcremon-meta — APPROVED** [review]
Looks good with a couple linter-related comments!

**Review by zonglinpeng — APPROVED** [review]

File: backends/cadence/fusion_g3/operators/op_quantize.cpp (hunk 3 above)
Comment (ckmadhira) [reply backends/cadence/fusion_g3/operators/op_quantize.cpp:806]: > Nit: missing new line at end of file

All the operator files do not have a new line at the end. Not having a new line at the end is not shown as linter error.

**Pull request merged**